armus/migrations.py

```diff
@@ -0,0 +1,55 @@
+import os
+import re
+import importlib
+from exceptions import RuntimeError
+
+
+class MigrationError(RuntimeError):
+    pass
+
+
+migration_filter = re.compile(r"^migration_\d+.*\.py$")
+
+
+def _get_migration_versions(path):
+
+    migration_files = [
+        filename for filename in os.listdir(path) if migration_filter.match(filename)
+    ]
+    return sorted(migration_files)
+
+
+def _find_unapplied_migrations(path, applied_migrations):
+    all_migrations = _get_migration_versions(path)
+    new_migrations = [
+        os.path.splitext(migration)[0] for migration in all_migrations if migration not in applied_migrations
```

Comment:
It seems like there might be an issue that you're only removing the filename extension here, but importing via the basename elsewhere.

If I pass in "package/migrations" as my path, then all the values stored in the DB will be "package/migrations/migration_NNN_do_something", but you'll be trying to import "migrations.migration_NNN_do_something".

Not sure if it needs to be changed, it just seems a little unexpected.